### PR TITLE
Adds `ribbon`

### PR DESCRIPTION
**Pull request by karsa-mistmere**
# Icon Request

* Icon name: ribbon
* Use case: raising awareness (e.g. breast cancer)
* Screenshots of similar icons:
  image

### CODE REVIEW

**Comment by ericfennis** [comment]
Hmm on 100% the icon is a bit dense. Maybe adding a small gap to the ribbon ends where they overlap?

**Comment by karsa-mistmere** [comment]
@ericfennis: I couldn't fit in large enough gaps that still left the design recognisable, it would be something like this:
image

What do you think about this one instead?
ribbon

**Comment by karsa-mistmere** [comment]
> The loop could also go back like this, feels a bit better to me.
> 
> icon (4)

Not bad, although also a bit too dense, we could also mix and match the two versions (left and right ones)?
image

File: icons/ribbon.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "awareness",
+    "strip",
+    "band",
+    "tape",
+    "strap",
```
Comment (jguddas) [inline icons/ribbon.json:11]:
```suggestion
    "strap",
    "scarf",
    "muffler",
```

File: icons/ribbon.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "awareness",
+    "strip",
+    "band",
+    "tape",
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/ribbon.json:10]: Grim and exhaustive list! 😬

[the remark this answers is not in the document]
Comment (ericfennis) [reply icons/ribbon.json:10]: @jguddas I think we can make this list shorter bij splitting up word by a space.
Filtering on the website mostly done by parts of a word and not by a combination of words.
So the list can be a bit reduced, example:
```diff
-"brain cancer",
-"brain disorders",
-"brain illness",
-"brain tumor",
-"kidney cancer",
-"kidney disease",
-"liver cancer",
-"lung disease",
+"brain"
+"kidney"
+"lung",
+"disease",
+"cancer",
+"illness",
+"tumor"
```

**Comment by ericfennis** [comment]
@karsa-mistmere That looks better!

**Review by ericfennis — APPROVED** [review]

File: icons/ribbon.svg [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M17.75 9.01c-.52 2.08-1.83 3.64-3.18 5.49l-2.6 3.54-2.97 4-3.5-2.54 3.85-4.97c-1.86-2.61-2.8-3.77-3.16-5.44" />
+  <path d="M17.75 9.01A7 7 0 0 0 6.2 9.1C6.06 8.5 6 7.82 6 7c0-3.5 2.83-5 5.98-5C15.24 2 18 3.5 18 7c0 .73-.09 1.4-.25 2.01Z" />
+  <path d="m9.35 14.53 2.64-3.31" />
+  <path d="m11.97 18.04 2.99 4 3.54-2.54-3.93-5" />
+  <path d="M14 8c0 1-1 2-2.01 3.22C11 10 10 9 10 8a2 2 0 1 1 4 0Z" />
```
Comment (jguddas) [inline icons/ribbon.svg:16]:
```suggestion
  <path d="M14 8c0 1-1 2-2.01 3.22C11 10 10 9 10 8a2 2 0 1 1 4 0" />
```

**Review by jguddas — APPROVED** [review]

**Pull request merged**